cmd/version/version.go

```diff
@@ -117,10 +117,14 @@ func checkForLatestVersion(currVersion string) {
 		return
 	}
 
+	
 	latestVersion := data["spec"].(map[string]interface{})["version"].(string)
 	if !isLatestVersion(currVersion, latestVersion) {
-		// The seperator for the error print
 		fmt.Println()
+		if currVersion == "dev" {
+			fmt.Printf("You are using development version of cli, latest released version is: %s", latestVersion)
+			return
+		}
 		fmt.Printf("You are using an older version (%s) of cli, latest available version is: %s", currVersion, latestVersion)
```

Comment:
Please convert it to `fmt.Println` so that a newline(the correct one depending upon the system) is added.
The newline is a confusing character across operating systems and sometimes it's best left for the libraries to deal with them correctly.
Ref: https://stackoverflow.com/a/3720674